plottable.js

```diff
@@ -5826,52 +5828,51 @@ var Plottable;
                 var swatchY;
                 var boundingBoxAttr = {
                     x: 0,
-                    y: padding,
+                    y: this._isVertical() ? 0 : padding,
                     width: 0,
                     height: 0
                 };
                 var numSwatches = InterpolatedColorLegend._DEFAULT_NUM_SWATCHES;
-                var textHeight = this._measurer.measure().height;
                 if (this.expands() && textHeight > 0) {
-                    var offset = this._isVertical() ? 2 * padding : 4 * padding - text0Width - text1Width;
+                    var offset = this._isVertical() ? 0 : 2 * padding - text0Width - text1Width;
```

Comment:
a vertical axis used to have padding on the top and bottom, so height - (2 * padding) was the height available to the swatches.

a horizontal axis used to have padding, text, padding, swatches, padding, text, padding. Now the outside padding is gone

that being said, i think that this should actually be 2 * textPadding - text0Width -text1Wdith, because textPadding is the (fixed) width between the text and swatches.